Northstar.Client/mod.json

```diff
@@ -130,6 +134,13 @@
 			"Path": "ui/ns_slider.nut",
 			"RunOn": "UI"
 		},
+		{
+            "Path": "ui/menu_ns_color_picker.nut.nut",
```

Comment:
crashes the game on start-up, change it to just ``.nut``

Reply:
oh my bad. I renamed the script, and forgot to check that. Thanks.